src/bucket/Bucket.cpp

```diff
@@ -859,6 +863,7 @@ Bucket::scanForEviction(AbstractLedgerTxn& ltx, EvictionIterator& iter,
     {
         if (be.type() == INITENTRY || be.type() == LIVEENTRY)
         {
+            ZoneNamedN(maybeEvict, "maybe evict entry", true);
```

Comment:
This zone isn't very meaningful here, since most entries we scan will not be temporary and will require almost no work. I think if we rename this to `should evict` and move this inside the lambda it would be more meaningful.

Reply:
sounds good. I just moved it inside the `if (isTemporaryEntry(le.data))` block